### PR TITLE
Create Websocket 2.2 API with 2.2.0-M1

### LINKED ISSUE #27036: Create io.openliberty.jakarta.websocket.2.2
Upload with M1 once released

### LINKED ISSUE #27037: Update io.openliberty.jakarta.pages.4.0 with 4.0.0-M1
Recently Released: https://repo1.maven.org/maven2/jakarta/servlet/jsp/jakarta.servlet.jsp-api/4.0.0-M1/

### PR DESCRIPTION
**Pull request by volosied**
fixes #27036

### CODE REVIEW

**Comment by pmd1nh** [comment]
do you need to check in cnf/dependabot/check_this_in_if_it_changes/pom.xml ?

File: dev/io.openliberty.jakarta/pages.4.0.bnd [hunk 1]
```diff
@@ -13,20 +13,20 @@ bVersion=1.0
 # This currently is a copy of the pages.3.1.bnd with the exception of the jakarta.el import version.
 # The jakarta.el import version needed to be updated to include 6.0 for Expression Language 6.0.
 
-javac.source: 17
```
Comment (pmd1nh) [inline dev/io.openliberty.jakarta/pages.4.0.bnd]: keep these 2 javac. at 17 for now?

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build

File: dev/io.openliberty.jakarta/websocket.client.2.2.bnd [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.websocket.client.2.2; singleton:=true
+
+javac.source: 17
+javac.target: 17
```
Comment (pnicolucci) [inline dev/io.openliberty.jakarta/websocket.client.2.2.bnd]: Update to 21

File: dev/io.openliberty.jakarta/websocket.server.2.2.bnd [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.websocket.2.2; singleton:=true
+
+javac.source: 17
```
Comment (pnicolucci) [inline dev/io.openliberty.jakarta/websocket.server.2.2.bnd]: Update to 21

**Review by pnicolucci — APPROVED** [review]

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#build

**Comment by volosied** [comment]
#libby

**Pull request merged**